test/widgets/user_test.dart

```diff
@@ -28,9 +27,12 @@ void main() {
       await store.addUser(user);
 
       prepareBoringImageHttpClient();
-      await tester.pumpWidget(GlobalStoreWidget(
-        child: PerAccountStoreWidget(accountId: eg.selfAccount.id,
-          child: AvatarImage(userId: user.userId, size: size ?? 30))));
+      await tester.pumpWidget(
+          TestZulipApp(
+            accountId: eg.selfAccount.id,
+            child: AvatarImage(userId: user.userId, size: size ?? 30),
+          )
+      );
```

Comment:
nit: keep indentation style (here and many other places):

```suggestion
      await tester.pumpWidget(
        TestZulipApp(accountId: eg.selfAccount.id,
          child: AvatarImage(userId: user.userId, size: size ?? 30)));
```